### ISSUE TITLE
Unable to login

### ISSUE THREAD

**Issue by tomasb82** [sindrebroch/ha-flexit#15]
When entering credentials geting error. What a problem can be?

![image](https://user-images.githubusercontent.com/74060061/146649683-74062e6c-ade0-43bc-aa7c-180076aac820.png)

**Comment by sindrebroch** [comment]
If you turn on debug-logs it might be easier to get an idea of what the problem is. 

Code to add to your configuration for debug-logging is at the bottom of the README

**Comment by tomasb82** [comment]
2021-12-20 20:09:58 DEBUG (MainThread) [custom_components.flexit] POST-request to url=https://api.climatixic.com/Token. data=grant_type=password&username=tomas@xxxxx.lt&password=xxxx. headers={'Accept': 'application/json', 'Accept-Encoding': 'gzip, deflate, br', 'Accept-Language': 'en-us', 'Content-Type': 'application/json; charset=utf-8', 'User-Agent': 'Flexit%20GO/2.0.6 CFNetwork/1128.0.1 Darwin/19.6.0', 'Ocp-Apim-Subscription-Key': 'c3fc1f14ce8747588212eda5ae3b439e'}
2021-12-20 20:09:58 DEBUG (MainThread) [custom_components.flexit] FlexitToken={'error': 'invalid_grant', 'error_description': 'The user name or password is incorrect.'}

**Comment by tomasb82** [comment]
I replased my credentilas with xxx, but they are fine as I can login to mobile app with them. Any Ideas?

**Comment by tomasb82** [comment]
Changed special charter form + to !. and it helped.

**Comment by sindrebroch** [comment]
Great

**Comment by kjartaor** [comment]
Thanks for great work! Had some username & password problems, but got it to wotk after I changed the password on my app. This was after I changed it on the website and that work . My procedure was to log out from the Flexit app, click forget/restore password and then apply the same password. I don't think it has to be the same, but it requiers you to have one that has minimum 8 characters, small and big letters and a special sign (!,?...). It also seems to me like it's a delay from changing the password on the site until its acuallay changed in the app. and it seems like  it suddenly works. Brilliant! It strange thoug, because I ended up changing the password back to the original.
